src/main/scala/org/ergoplatform/ErgoBoxCandidate.scala

```diff
@@ -51,11 +52,12 @@ class ErgoBoxCandidate(val value: Long,
     }
   }
 
-  override def hashCode() = ScalaRunTime._hashCode((value, proposition, additionalTokens, additionalRegisters))
-
+  override def hashCode(): Int =
+    ScalaRunTime._hashCode((value, proposition, additionalTokens, additionalRegisters, creationHeight))
```

Comment:
@kushti It is better to avoid allocations in hashCode. Because this method is usually hotspot and can be called many times during object lifecycle.

Reply:
@ergomorphic what to do instead, use Statics.mix directly instead? Btw, I realized that there's no hashCode impl for Value, probably it should be implemented?